src/index.ts

```diff
@@ -177,13 +178,13 @@ async function replaceAsync(str: string, data: Data) {
 function genProperties(properties) {
     const { assignee, priority, fixVersion, status, reporter, summary, resolution } = properties;
     const { showAssignee,
-        showPriority, 
-        showFixVersion, 
-        showStatus, 
+        showPriority,

```

Comment:
@adyscorpius VS Code automatically fixed the whitespace errors.
If you want me to take those changes out let me know 😃

Reply:
No worries, that's great. Is there a way I could test with an on premise instance of Jira without necessarily buying a license? It'll help make sure it works okay.

Reply:
It might work with the following approach:
https://community.atlassian.com/t5/Jira-articles/How-to-run-Jira-in-a-docker-container/ba-p/752697

Reply:
@nightscape  thanks for this work, but I'm hitting an issue:

![image](https://github.com/adyscorpius/logseq-jira/assets/5154224/55ce5271-894d-44af-8994-fc3e7fd3d720)

Reply:
Can you open the Network Inspector in your browser and see what the response message in this 403 is?
And can you try using your password instead of your token as password? For me the token does not work and gives a 403, but the password does.

Reply:
ah, thanks my instance uses corporate SSO, so no password available, no worries, I can live without it.

Reply:
It might be worth opening the Network inspector in your browser while browsing the JIRA page. You should see requests to the REST API and could check how the authentication is handled there.

Reply:
I am interested in the integration with hosted jira, so I've tested the code on this pull request.

I have the same error, and for me it's because jira doesn't consider my credentials to be valid. Maybe a first improvement would be to check the status code of the response? When it's 401, the user is not authenticated.

For hosted jira, it seems that when using a personal access token, it must be provided as a bearer token instead of basic authentication. This change works for me:
```typescript
'Authorization': `Basic ${creds}`
```
to
```typescript
'Authorization': `Bearer ${logseq.settings?.jiraAPIToken}`
```

Maybe this could help you?

Reply:
@pwright can you test?

Reply:
@nightscape I've gone with a different approach, not interested in a logseq plugin any more... but I think the comment about bearer token looks correct